eng/pipelines/templates/common.yml

```diff
@@ -1,22 +1,28 @@
 parameters:
-- name: PublishPackages
+- name: ServerName
+  type: string
+  default: ''
+- name: PublishTarget
+  type: string
+  default: none
+  values:
+  - internal
+  - public
+  - none
+- name: RunLiveTests
   type: boolean
   default: false
-- name: ReleaseRun
+- name: IncludeNative
   type: boolean
   default: false
-- name: RunLiveTests
+- name: IncludeDockerRelease
```

Comment:
Just want to make sure, the verbiage flipped from exclusion to inclusion, but the value was kept as `false`, was that intended?

Reply:
Yes.. I want to make docker release opt in, but I went ahead and set values in all the servers' build.yml files